ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -213,3 +213,23 @@ def csingle(x):
 @to_ivy_arrays_and_back
 def cdouble(x):
     return ivy.astype(x, ivy.complex128)
+
+
+@to_ivy_arrays_and_back
+def compress(condition, a, *, axis=None):
```

Comment:
This is not exactly the signature of the native function. There is also an `out` argument you will need to add and handle.